ragna/deploy/_ui/central_view.py

```diff
@@ -235,19 +234,37 @@ def on_click_chat_info_wrapper(self, event):
             ],
         )
 
-    def on_click_source_info_wrapper(self, event, sources):
+    async def on_click_source_info_wrapper(self, event, sources):
```

Comment:
Why did this change to `async`?

Reply:
I spent some time trying to make this work as a synchronous callback but was unsuccessful. Since the call to api was async it was throwing a lot of complaints but its back to being synchronous again now that its a JS callback so I'll see if it complains again!